client/components/button/style.scss

```diff
@@ -76,6 +76,14 @@ button {
 			top: 4px;
 			margin-top: -8px;
 		}
+		// Make the left margin of the small plus icon visually less huge
+		.gridicons-plus-small {
```

Comment:
I could add something like this next to that

`.gridicon-plus-small:last-child { margin-left: 0; }`

To set it back to 0 if there was not a gridicon next to it, but that's obviously a little ugly. Thoughts @mtias or @MichaelArestad ?